KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutPresenter.swift

```diff
@@ -196,18 +196,22 @@ final class KarhooCheckoutPresenter: CheckoutPresenter {
     }
 
     // MARK: - Book
-    func bookTripPressed() {
-        guard areTermsAndConditionsAccepted else {
-            self.view?.showTermsConditionsRequiredError()
+    func completeBookingFlow() {
```

Comment:
The flow is as before. The user needs to click the "Book Now" button after authorising the card. The scope of the ticket is not covered.

Reply:
After some deep cleaning the feature started working as expected. Everything looks ok now.